static_analysis.bash

```diff
@@ -5,16 +5,19 @@ npx eslint \
     'src/composables/**/*.ts' \
     'src/components/validated_input/**/*.vue' \
     'src/demos/new_validated_input_demo/**/*.vue' \
+    'src/demos/new_validated_form_demo/**/*.vue' \
+    'tests/test_composables/**/*.ts' \
     'tests/test_components/test_new_validated_input.ts' \
     'tests/test_components/test_new_validated_form.ts'
 
-npx prettier --check \
-    './*.js' \
-    './*.ts' \
+npx prettier --check --no-editorconfig \
```

Comment:
.editorconfig specifies indent size for different files, so for now I'm just ignoring it when running prettier. Alternatively, we can add new files that should use indent size of 2 in .editorconfig. I don't love the idea of adding each new file to two places in static_analysis.bash and .editorconfig though.